planner/core/planbuilder.go

```diff
@@ -2488,14 +2486,17 @@ func calculateTsExpr(sctx sessionctx.Context, asOfClause *ast.AsOfClause) (uint6
 	return oracle.GoTimeToTS(tsTime), nil
 }
 
-func collectVisitInfoFromRevokeStmt(sctx sessionctx.Context, vi []visitInfo, stmt *ast.RevokeStmt) []visitInfo {
+func collectVisitInfoFromRevokeStmt(sctx sessionctx.Context, vi []visitInfo, stmt *ast.RevokeStmt) ([]visitInfo, error) {
 	// To use REVOKE, you must have the GRANT OPTION privilege,
 	// and you must have the privileges that you are granting.
 	dbName := stmt.Level.DBName
 	tableName := stmt.Level.TableName
 	// This supports a local revoke SELECT on tablename, but does
 	// not add dbName to the visitInfo of a *.* grant.
 	if dbName == "" && stmt.Level.Level != ast.GrantLevelGlobal {
+		if (sctx.GetSessionVars().CurrentDB == "") {
```

Comment:
paren is not needed. Please fix CI.

Reply:
Please also run `go fmt planner/core/planbuilder.go` (although this might be the issue).

Otherwise LGTM.

Reply:
@xhebox done.
@morgo run make testpkg pkg=planner/core, it passed.